### PR TITLE
add os-test tests

### PR DESCRIPTION
**Pull request by jabia99**
test_no_sshkeys_knownhosts -> test_no_sshkeys_knownhosts

### CODE REVIEW

**Comment by jabia99** [comment]
This test is passing, should we merge now or wait until we have the official images list?

File: test_suite/generic/test_generic.py [hunk 1]
```diff
@@ -310,6 +310,16 @@ def test_rpm_v_unsatisfied_dependencies(self, host):
             assert 'Unsatisfied' not in host.run('rpm -Va').stdout, \
                 'There are unsatisfied dependencies'
 
+    @pytest.mark.run_on(['all'])
+    def test_no_sshkeys_knownhosts(self, host):
+        '''
```
Comment (narmaku) [inline test_suite/generic/test_generic.py:315]: Let's use double quotes here to be consistent.

```suggestion
        """
```

File: test_suite/generic/test_generic.py [hunk 2]
```diff
@@ -310,6 +310,16 @@ def test_rpm_v_unsatisfied_dependencies(self, host):
...
+        Verify no extra files under /root/.ssh/ except authorized_keys
+        '''
+        with host.sudo():
+            ssh_files = host.file('/root/.ssh/').listdir()
+            assert 'authorized_keys' in ssh_files, 'authorized_keys is not in /root/.ssh/'
+            assert len(ssh_files) == 1, 'there is extra files in /root/.ssh/'
```
Comment (narmaku) [inline test_suite/generic/test_generic.py:321]:
```suggestion
            assert len(ssh_files) == 1, 'There are extra files in /root/.ssh/'
```

**Review by narmaku — APPROVED** [review]
Congrats on the first os-tests officially migrated! I added only a couple of comments.

**Review by narmaku — APPROVED** [review]

**Pull request merged**